### PR TITLE
Add account plans controller

### PR DESCRIPTION
**Pull request by msimionati**
add new account plans controller and adapt existing tests to it, change old project name references and restore weather controller for test purpose.

### CODE REVIEW

File: Doppler.AccountPlans.Test/IntegrationTest1.cs [hunk 1]
```diff
@@ -18,10 +18,9 @@ public IntegrationTest1(WebApplicationFactory<Startup> factory)
         }
 
         [Theory]
+        [InlineData("/WeatherForecast", HttpStatusCode.OK, "application/json; charset=utf-8")]
         [InlineData("/swagger", HttpStatusCode.Moved, null)]
         [InlineData("/swagger/index.html", HttpStatusCode.OK, "text/html; charset=utf-8")]
-        [InlineData("/robots.txt", HttpStatusCode.OK, "text/plain")]
-        [InlineData("/favicon.ico", HttpStatusCode.OK, "image/x-icon")]
```
Comment (andresmoschini) [inline Doppler.AccountPlans.Test/IntegrationTest1.cs:24]: `favicon.ico` and `robots.txt` should work, why did you remove them?

File: Doppler.AccountPlans/Controllers/AccountPlansController.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+using Microsoft.Extensions.Logging;
+
+namespace Doppler.AccountPlans.Controllers
+{
+    [Authorize]
+    [ApiController]
+    public class HelloController
+    {
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        [HttpGet("/accounts/{accountName}/newplan/{planId}/calculate/{promocode?}")]
```
Comment (andresmoschini) [inline Doppler.AccountPlans/Controllers/AccountPlansController.cs:17]: Sinse promocode is optional, I think that it should be part of the querystring

```
 ACTUAL: https://apis.fromdoppler.com/plans/accounts/amoschini@makingsense.com/newplan/123/calculate/PROMO-123`
MY PREF: https://apis.fromdoppler.com/plans/accounts/amoschini@makingsense.com/newplan/123/calculate?promocode=PROMO-123`
```
```suggestion
        [HttpGet("/accounts/{accountName}/newplan/{planId}/calculate")]
```

File: Doppler.AccountPlans/Controllers/AccountPlansController.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        public string GetCalculateUpgradeCost([FromRoute] string accountName, [FromRoute] int planId, [FromRoute] string promocode = "doppler")
```
Comment (andresmoschini) [inline Doppler.AccountPlans/Controllers/AccountPlansController.cs:18]: why do you have a promocode by default?
```suggestion
        public string GetCalculateUpgradeCost([FromRoute] string accountName, [FromRoute] int planId, [FromQuery] string promocode = null)
```

File: README.md [hunk 4]
```diff
@@ -1,123 +1,3 @@
-# hello-microservice
+# doppler-account-plans
```
Comment (andresmoschini) [inline README.md:1]: It is out of the scope of this PR (_"Add account plans controller"_).

I can live with that, but do not forget to search other places where `hello` have not been replaced, for example, `.vscode/tasks.json`

File: Doppler.AccountPlans.Test/IntegrationTest1.cs (hunk 1 above)
Comment (msimionati) [reply Doppler.AccountPlans.Test/IntegrationTest1.cs:24]: ok, I was using doppler-users-api as an example to follow and they weren't there, that's why I removed them

Comment (andresmoschini) [reply Doppler.AccountPlans.Test/IntegrationTest1.cs:24]: Oh, @javierburger please restore them in your API.

Comment (javierburger) [reply Doppler.AccountPlans.Test/IntegrationTest1.cs:24]: ok

File: Doppler.AccountPlans/Controllers/AccountPlansController.cs (hunk 3 above)
Comment (msimionati) [reply Doppler.AccountPlans/Controllers/AccountPlansController.cs:18]: you're right, it should be null by default. As there is another task that is to create the endpoint itself, I did not think about it too much.
I change it.

File: README.md (hunk 4 above)
Comment (andresmoschini) [reply README.md:1]: Done! resolving the conversation.
